scripts/ansible-ssh.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+"""
+CDT-2221 Alpha Red Team
+Author: Anthony Swierkosz; ajs2576
+
+Installs SSH keys for Ansible to use.
+"""
+
+import subprocess
+
+user = "Administrator"
+ip = "10.1.4.7"
+
+
+def windows_ssh(user, ip):
+    print(f'\n\n{"":-^48}')
+    print(f'{f" {user}@{ip} ":-^48}')
+    print(f'{"":-^48}')
+
+    subprocess.run(
+        ' '.join(['sudo', 'scp', '~/.ssh/id_rsa.pub', f'{user}@{ip}:/ProgramData/ssh/administrators_authorized_keys']),
+        shell=True)
+
+    print(f"Setting permissions for '{user}@{ip}'...")
+    subprocess.run(' '.join(['sudo', 'ssh', f'{user}@{ip}',
+                             '\'icacls.exe "C:\ProgramData\ssh\\administrators_authorized_keys" /inheritance:r /grant "Administrators:F" /grant "SYSTEM:F"\'']),
+                   shell=True)
+
+    subprocess.run(' '.join(['sudo', 'ssh', f'{user}@{ip}',
+                             '\'icacls "C:\ProgramData\ssh\\administrators_authorized_keys" /remove "NT AUTHORITY\Authenticated Users"\'']),
+                   shell=True)
+
+    subprocess.run(' '.join(['sudo', 'ssh', f'{user}@{ip}',
+                             '\'icacls "C:\ProgramData\ssh\\administrators_authorized_keys" /inheritance:r\'']),
+                   shell=True)
```

Comment:
This is very rudimentary and requires user interaction which is annoying. Would be nice if this could automatically run and handle all required user input (e.g., known hosts yes/no, SSH password, etc.). Maybe also move to `subprocess.popen`?

Reply:
Being handled by https://github.com/aisgbnok/Ansible-2221/issues/8#issue-1450977174.